STYLEGUIDE.md

```diff
@@ -20,6 +20,10 @@ It is dangerous to assume that because something is obvious to you, it will also
 
 ### Additional Suggestions
 
+**Use American English**
+
+PostHog is an American company. As a result, while our Documentation is only in one language, we would prefer to keep it consistent by using American spelling for words.
```

Comment:
I don't agree we're an american company. I'd rephrase as "We're a global company, but to keep our communication consistent we use American spelling"

Reply:
Right, that makes sense. I was basing that on James' comment but this makes perfect sense. American company does not sound good.

Reply:
Especially given the distributed team

Reply:
Agreed, my comment wasn't really phrased well. We have a US parent entity, so that's the natural choice here